testing/run_tests.sh

```diff
@@ -6,7 +6,7 @@ out/host_debug_unopt/fml_unittests
 out/host_debug_unopt/synchronization_unittests
 
 pushd flutter/testing/dart
-pub get
+flutter pub get
```

Comment:
What is the reason for this change?

I'd like to avoid having the engine tree depend on the framework tree

Reply:
Discussed offline, removed.